payment-failure/src/main/scala/com/gu/identity/paymentfailure/Model.scala

```diff
@@ -1,15 +1,36 @@
 package com.gu.identity.paymentfailure
 
-import io.circe.generic.semiauto.deriveDecoder
-import io.circe.Decoder
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class Config(idapiHost: String, brazeApiHost: String, idapiAccessToken: String, queueURL: String, brazeApiKey:String)
 
 case class IdentityBrazeEmailData(externalId: String, emailAddress: String, templateId: String, customFields: Map[String, String])
 
 object IdentityBrazeEmailData {
   implicit val identityBrazeEmailDataDecoder: Decoder[IdentityBrazeEmailData] = deriveDecoder[IdentityBrazeEmailData]
 }
 
-case class BrazeResponse(msg: String)
+case class BrazeRecipient(external_user_id: String, trigger_properties: Option[Map[String, String]])
+
+object BrazeRecipient {
+  implicit val brazeRecipientEncoder: Encoder[BrazeRecipient] = deriveEncoder[BrazeRecipient]
+}
 
-case class Config(idapiHost: String, brazeApiHost: String, idapiAccessToken: String, queueURL: String)
+case class BrazeSendRequest(api_key: String,
+                            campaign_id: String,
+                            trigger_properties: Option[Map[String, String]],
```

Comment:
Is the `trigger_properties` field required?

Reply:
This is what Braze uses for custom strings we want to pass to the template. For example here we are adding the encryptedEmail to the trigger_properties, we will then access it in the braze template html.